SYCL/lit.cfg.py

```diff
@@ -281,6 +281,20 @@
 if find_executable('sycl-ls'):
     config.available_features.add('sycl-ls')
 
+llvm_spirv_path = find_executable("llvm-spirv")
```

Comment:
why these checks are added here. As far AFAIR these tools are always present in the compiler. So the check is redundant.

In any case, I do not see any usages of the substitutions defined here

Reply:
`llvm-spirv` and `llvm-link` are used in SYCL/AOT/multiple-devices.cpp

Reply:
The tools do but substitutions that have been defined don't.

Reply:
> but substitutions that have been defined don't

It's explicitly `%LLVM_LINK` and `%LLVM_SPIRV` which are used now in `multiple-devices.cpp`

> REQUIRES: lvm-spirv

is there already a feature defined as `llvm-spirv` or `llvm-link`? Is there any simple example of defining such a feature?

Reply:
you can use sycl-ls as example

Reply:
feature-related comment still not applied

Reply:
Fixed.